### DISCUSSION TITLE
Restricted to premium users

### DISCUSSION THREAD

**Discussion by MayankPunghal** [gilbarbara/react-spotify-web-playback#95] [Q&A] [answered]
The web player is now restricted for the use of premium users only. Is there any free alternative to this player?

**Comment by gilbarbara** [comment, marked as answer]
This restriction always existed since non-premium users can only play the public preview file.

**Reply by MayankPunghal** [reply]
Thanks for the answer. Although I was wondering is there any other alternative or a solution for the non premium users?
